assets/scss/_legacy.scss

```
@@ -0,0 +1,6 @@
+.legacy {
+  margin-top: 6rem;
+  @include media-breakpoint-down(sm) {
+    margin-top: 2rem;
+  }
+}
```

```suggestion
}
img[alt="Flux v1 Diagram"] {
  width: 100%;
}
```

If any image is used anywhere with this alt text, it is set to 100% width. This fixes the "diagram is huge" issue and hopefully doesn't affect anything else. (I'm not sure how to make changes that only apply to the legacy docs, if it is sufficient to put them in this file, or if there are other cases, maybe we should simply apply this formatting to every image in the legacy docs.)

---

Looks like there is a `max-width: 100%` applied to images in the "current" docs website from somewhere. This is the more elegant way (but I don't see where it's been set from as of yet, or how to repeat it in the "legacy" docs.)

---

I think it must come from:


```
// Responsive image
//
// Keep images from scaling beyond the width of their parents.

@mixin img-fluid() {
  // Part 1: Set a maximum relative to the parent
  max-width: 100%;
  // Part 2: Override the height to auto, otherwise images will be stretched
  // when setting a width and height attribute on the img element.
  height: auto;
}
```

`themes/docsy/assets/vendor/bootstrap-4.6.0/scss/mixins/_image.scss`

If there is a simple change that will include all of the "mixins" such as this one that apparently resolves this issue in `current` docs section, that will probably correct a lot of the issues that I'm identifying with my read-through now.

---

We need to make sure `type: legacy` has the same styling as `type: docs`.